custom_components/alexa_media/alexa_entity.py

```diff
@@ -263,7 +263,9 @@ def parse_alexa_entities(network_details: Optional[dict[str, Any]]) -> AlexaEnti
     air_quality_sensors = []
     contact_sensors = []
     switches = []
-    location_details = network_details["locationDetails"]["locationDetails"]
+    location_details = (
+        network_details["locationDetails"]["locationDetails"] if network_details else {}
```

Comment:
Might be best to also do a double get of locationDetails with a default of {} in case the nested dict is malformed.

Reply:
Done